### PR TITLE
migration: Updates migration of internal_location

### PR DESCRIPTION
**Pull request by jrcastro2**
* legacy_id can be an array or an integer
* Blocked by release of [invenio-app-ils](https://github.com/inveniosoftware/invenio-app-ils/pull/1015)
* closes https://github.com/CERNDocumentServer/cds-ils/issues/280

### CODE REVIEW

**Review by KonstantinaStoikou — APPROVED** [review]

File: cds_ils/migrator/acquisition/vendors.py [hunk 1]
```diff
@@ -29,11 +29,29 @@
...
+    # Check for Wiley vendor to split it depending on the currency
+    if legacy_id == ORIGINAL_WILEY_ID:
+        if grand_total and grand_total['currency']:
+            legacy_id = WILEY_MAPER.get(grand_total['currency'], DEFAULT_WILEY)
+        else:
+            legacy_id = DEFAULT_WILEY
+
     search = current_ils_acq.vendor_search_cls().filter(
-        "term", legacy_id=legacy_id
+        "term", legacy_ids=legacy_id
```
Comment (kprzerwa) [inline cds_ils/migrator/acquisition/vendors.py:54]: is it tested? I had many problems with the searched after we changed mappings, so I'm asking to be extra sure

File: cds_ils/migrator/acquisition/vendors.py [hunk 2]
```diff
@@ -56,6 +74,8 @@ def import_vendors_from_json(dump_file):
     with click.progressbar(json.load(dump_file)) as input_data:
         ils_records = []
         for record in input_data:
+            if not isinstance(record["legacy_ids"], list):
```
Comment (kprzerwa) [inline cds_ils/migrator/acquisition/vendors.py:77]: could you leave a comment here why we are doing this?

File: cds_ils/migrator/acquisition/vendors.py (hunk 1 above)
Comment (jrcastro2) [reply cds_ils/migrator/acquisition/vendors.py:54]: Yes, here is the proof:
image

Comment (kprzerwa) [reply cds_ils/migrator/acquisition/vendors.py:54]: thanks a lot :)

**Review by kprzerwa — APPROVED** [review]

**Pull request merged**